.github/workflows/dev-docker-image-build.yml

```diff
@@ -27,4 +27,6 @@ jobs:
             # Pull, retag and push to GitHub packages
             docker buildx build --platform='linux/amd64,linux/arm64' $GITHUB_TAG --push --label "org.opencontainers.image.revision"=${{ github.sha }} .
             
-            docker buildx prune -a -f --filter until=12h
```

Comment:
this prune step only cleared the build-cache which seems to be always 0 in our case.